.python-version

```diff
@@ -0,0 +1 @@
+3.8.1
```

Comment:
ignore this file

Reply:
If you do this frequently to projects, it's probably a good idea to put it in your global .gitignore rather than committing it and pushing it as part of the PR.

Reply:
Actually I usually want it checked in. Just not in this case.

Reply:
(Removed the noise though)